test/apiv2/35-networks.at

```diff
@@ -6,9 +6,8 @@
 t GET networks/non-existing-network 404 \
   .cause='network not found'
 
-# FIXME FIXME FIXME: failing in CI. Deferring to someone else to fix later.
 #if root; then
-if false; then
+if root; then
```

Comment:
This check can be removed. We now support rootless networking. I tried running this as user and it worked.